### PR TITLE
feat: add zendesk widget

### PR DESCRIPTION
**Pull request by leangseu-edx**
Screen Shot 2022-11-17 at 11 32 14 AM
Screen Shot 2022-11-17 at 11 32 08 AM

### CODE REVIEW

File: src/components/ZendeskFab.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { getConfig } from '@edx/frontend-platform';
+
+import Zendesk from 'react-zendesk';
+
+// Take contact form as an example
```
Comment (jansenk) [inline src/components/ZendeskFab.jsx:7]: Was this copy pasted?

Comment (leangseu-edx) [reply src/components/ZendeskFab.jsx:7]: Yes 😸

File: src/components/ZendeskFab.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      },
+      attachments: true,
+    },
+    helpCenter: {
+      originalArticleButton: true,
+    },
+    answerBot: {
+      suppress: false,
+      contactOnlyAfterQuery: true,
+      title: { '*': 'edX Support' },
```
Comment (muselesscreator) [inline src/components/ZendeskFab.jsx:35]: plz internationalize

**Review by muselesscreator — APPROVED** [review]
minor translation request

**Pull request merged**